Core/src/org/sleuthkit/autopsy/mainui/nodes/TreeChildFactory.java

```diff
@@ -217,4 +251,6 @@ protected void addNotify() {
      * @throws ExecutionException
      */
     protected abstract TreeResultsDTO<? extends T> getChildResults() throws IllegalArgumentException, ExecutionException;
+
+    protected abstract TreeItemDTO<? extends T> getInvalidatedChild(TreeEvent daoEvt);
```

Comment:
Needs a comment.  It looks like it needs to return something if the event is for a new child that doesn't exist.  If so, perhaps it should be something like getOrCreateRelevantChild()??